README.md

```diff
@@ -12,10 +14,92 @@ npm install --save "git://github.com/ditrit/leto-modelizer-plugin-core.git"
 
 ### Usage
 
+Here is all import you can use in your plugin :
+
+```js
+import {
+  Component,
+  ComponentDefinition,
+  ComponentAttributeDefinition,
+  ComponentDrawOption,
+  ComponentAttribute,
+  ComponentLink,
+  DefaultDrawer,
+  DefaultMetadata,
+  DefaultParser,
+  DefaultRender,
+  ParseError
+} from "leto-modelizer-plugin-core";
+```
+
+For more information about all imports please refer to technical documentation and project template.
+
+You can use it in that way:
+
 ```js
-import { renderString, LetoObjectNode } from "leto-modelizer-plugin-core";
+import { DefaultParser } from 'leto-modelizer-plugin-core';
+
+class FruitParser extends DefaultParser {
+  parse(inputs) {
+    // Write your own parser here
+    return {
+      components: [/* ... */], // Generated components from parse
+      links: [/* ... */]       // Generated component links from parse
+    }
+  }
+}
+
+export default FruitParser;
 ```
 
+To see complete example, please refer to [the fruit example](https://github.com/ditrit/leto-modelizer-plugin-template/tree/main/examples/fruit-plugin) in project template.
+
+## How to create your plugin
+
+You can use [leto-modelizer-plugin-template](https://github.com/ditrit/leto-modelizer-plugin-template) project as template to create your own plugin.
+
+The project template leto-modelizer-plugin-template provides you the default structure of a plugin and all useful scripts to generate resources and other.
+
+It is strongly recommended to use it and all next explanations will make reference to it.
```

Comment:
It is strongly recommended to use it and the following documentation will make references to it.